### PR TITLE
Feature/about us 2: Added changes for about us page.

### PR DESCRIPTION
**Pull request by irving-frias**
*  Finished section 4 block our benefits for the about us page.
* Added files about-us for less.
* Created styles for section 4 to 6, at about us page.

### CODE REVIEW

File: inc/about-us/section-5.inc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <div class="col">
+    
+    </div>
+
+    <div class="col">
+    
+    </div>
+
+  </div>
+</div>
```
Comment (jlsanchezfr) [inline inc/about-us/section-5.inc]: Please add newline at the end of the file.

File: inc/about-us/section-6.inc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <div class="col">
+    
+    </div>
+
+    <div class="col">
+    
+    </div>
+
+  </div>
+</div>
```
Comment (jlsanchezfr) [inline inc/about-us/section-6.inc]: Please add newline at the end of the file.

File: less/components/about-us/03-section-4.less [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#our-benenfits{
+  background-color: aqua;
+}
```
Comment (jlsanchezfr) [inline less/components/about-us/03-section-4.less]: Please add newline at the end of the file.

File: inc/about-us/section-4.inc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <p></p>
+      </div>
+
+      <div class="col-12">
+        <a href="">Read More</a>
+      </div>
+
+    </div>
+  </div>
+</div>
```
Comment (jlsanchezfr) [inline inc/about-us/section-4.inc]: Please add newline at the end of the file.

File: inc/about-us/section-5.inc (hunk 1 above)
Comment (irving-frias) [reply inc/about-us/section-5.inc]: ready

**Pull request closed without merging**